src/components/Layout/Layout.scss

```diff
@@ -1,8 +1,23 @@
 .layout {
   display: grid;
-  grid-template-columns: 25% auto;
+  grid-template-columns: .25fr .25fr .25fr .25fr;
   max-width: 1440px;
   margin: 0 auto;
-  padding: 0 16px;
+  padding: 16px 16px;
```

Comment:
I dont think we should add this to the top and bottom since this is now going to be a wrapper we should remove padding instead